tethys_portal/settings.py

```diff
@@ -392,3 +395,37 @@
 # Add any additional specified settings to module
 for setting, value in portal_config_settings.items():
     setattr(this_module, setting, value)
+
+# Cookie Settings
```

Comment:
Why are these commented lines here? Do we need to make a way to add these settings through the the `portal_config.yml` file?

Reply:
These are recommended security settings to use for the react setup (from this tutorial https://testdriven.io/blog/django-spa-auth/). However, restricting the CSRF to HTTPONLY means that it cannot be accessed through JavaScript, which a lot of apps take advantage of. This needs more research on our part, so I removed the commented-out lines and added an Issue to look into it: #836